Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:20 
+
+WORKDIR /app
+
+COPY package.json .
+COPY pnpm-lock.yaml .
```

@Srhiulli Perfeito aqui ❤️ , só mudaria o `COPY` e copiaria tudo em uma camada só, exemplo:

```suggestion
WORKDIR /app

COPY package.json pnpm-lock.yaml ./
```

Motivo disso: Quanto menos camadas no `docker` melhor e também um pouco de `cache`